### PR TITLE
Update cellx autoupdate config

### PR DESCRIPTION
**Pull request by andrew-anguiano**
Pull request for issue: #11932 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/Riim/cellx
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/cellx
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 9
   - Count of stars: 203
   - Count of forks: 13
 * NPM download stats (optional):
   - Downloads in the last day: 212
   - Downloads in the last week: 4,021
   - Downloads in the last month: 13,354

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/webvr-polyfill/package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
```
Comment (PeterDaveHello) [inline ajax/libs/webvr-polyfill/package.json]: This file seems to be unrelated with this pull request?

File: ajax/libs/cellx/package.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
     {
       "basePath": "dist",
       "files": [
-        "**/*"
+        "cellx.min.js"
```
Comment (PeterDaveHello) [inline ajax/libs/cellx/package.json]: So the new auto-update config will only fetch one file? But looks like you added multi assets?

**Comment by andrew-anguiano** [comment]
Yeah, something weird happened with my commit. Fixed it and tried to rebase but my computer freaks out and ends up pulling 80gb when I do that... I’ll check it out again and see if I can correct it. Otherwise, I may close this and start a new PR.

________________________________

**Comment by PeterDaveHello** [comment]
@andrew-anguiano see if sparseCheckout is enabled properly first. I can help you rebase the branch if you got  some problem.

**Comment by andrew-anguiano** [comment]
@PeterDaveHello I have sparse checkout enabled. I pulled down master with a depth of 1, switched to my `update-cellx-autoupdate` branch and ran `git pull master --rebase --depth 1` and basically git just hangs with the message `"First, rewinding head to replay your work on top of it..."`.

**Comment by PeterDaveHello** [comment]
@andrew-anguiano if it use so much space, it might simply mean that sparseCheckout is not properly configured,  may I see your output of command `git config core.sparseCheckout` and the content of `.git/info/sparse-checkout` at the root of your cdnjs repository? If you properly setup that, it should use less than 4GB :)

**Comment by andrew-anguiano** [comment]
@PeterDaveHello Sure!

`git config core.sparseCheckout    true`
```
/build
/documents
/test
/spec
/tools
/LICENSE
/hooks
/*.yml
/*.md
/*.json
/*.sh
/*.js
/.jscsrc
/ajax/libs/cellx/*
```

I just used the example given in the sparsecheckout info and replaced the lib with cellx. Thanks for your help!

**Comment by PeterDaveHello** [comment]
@andrew-anguiano looks good, try "git reset --hard" to see if it works?

**Comment by sufuf3** [comment]
@andrew-anguiano thanks for contribution.
Would you like to work on this and help improve this PR?
If you have any question, please let us know.

Many thanks.

**Comment by sufuf3** [comment]
@andrew-anguiano thanks for contribution.
Would you mind working on this and helping improve this PR?
If you have any question, please let us know.

Many thanks.

**Comment by PeterDaveHello** [comment]
I'll close this PR due to no response, we can re-open it if anytime @andrew-anguiano would like to :)
